Runner.py

```diff
@@ -0,0 +1,31 @@
+"""
+Оформите решение задач из прошлых домашних работ в функции.
+Напишите функцию runner.
+(все станет проще когда мы изучим модули, getattr и setattr)
+a.	runner() – все фукнции вызываются по очереди
+b.	runner(‘func_name’) – вызывается только функцию func_name.
+c.	runner(‘func’, ‘func1’...) - вызывает все переданные функции
+
+"""
+import funcs
+
+
+def runner(*args):
+    # print(*args)
+    if len(args) == 0:
```

Comment:
can without implemented without code duplication

Reply:
complete duplication of the code cannot be avoided (partially solved the problem by using the "check_name" function), since we check the conditions for two cases: there are no arguments or there are 1 or more